src/components/view/SignUp/index.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import {
+  Container,
+  Ads,
+  FormContainer,
+  Title,
+  InputGroup,
+  InputRowGroup,
+  InputItem,
+  InputTitle,
+  InputSection,
+  Submission,
+} from './styles';
+// import * as imgBack from 'assets/imgs/imgBack.jpg';
+import { Link } from 'react-router-dom';
+import { PATH } from 'consts';
+
+export const SignupView: React.FC = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [confirmpassword, setConfirmpassword] = useState('');
+  const [phone, setPhone] = useState('');
+  const [code, setCode] = useState('');
+  const [username, setUsername] = useState('');
```

Comment:
please use object for the state
`{email: '', password: '', ...}`
and you can handle all the input onchange event in one function. e.g.
`const onChange = (e) => {
   setState({[e.target.name]: e.target.value, ...state});
}`